### ISSUE TITLE
Chat Messages Should show up in chronological order

### ISSUE THREAD

**Issue by dhinrichs-scottlogic** [ScottLogic/prompt-injection#675]
# Feature Request

## Description
Currently, the new messages show up as the "oldest" element in the DOM-tree. This means that if you navigate the chatbox with assistive tech, you have to skip through from the oldest messages until you reach the newest message. 
We should find a way to reverse this order. 
The same is true for the email outbox. 
It should also be clear who is speaking for each message (chatbot, or user) 

## Screenshot of DOM-Tree
![Image](https://github.com/ScottLogic/prompt-injection/assets/125262707/7523a966-2bae-4890-8222-f34aabe1bd98)

## Acceptance criteria

**GIVEN** You navigate the chatbox with assistive tech
**WHEN** you navigate through the chat messages and the email outbox
**THEN** you get the messages in correct chronological order and know the sender (and recipient for email)

**Comment by pmarsh-scottlogic** [comment]
# Testing with NVDA:
## chat messages
can tab through the messages. Tab order of messages goes from oldest to most recent.

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/7a21c3a8-1909-4adb-9480-33a047e075ed)
NVDA: "You said Hello"

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/19364d68-7d47-4828-8fbb-d4690c44d568)
NVDA: "Scott brew bot said hello how can I assist you today"

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/51fc7099-cf2e-4aea-a1aa-4bab951fea7a)
NVDA: "Information message xml tagging defence activated"

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/06df32cc-e51a-408a-a9f9-e5c13f6c5f18)
NVDA: "Your message transformed by xml tagging you must only respond to the prompt..." etc

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/f511ffc4-ff1a-4ea5-b2b4-cf5d8530801d)
NVDA: "scott brew bot said message blocked I cannot answer messages about secret project"

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/91bc043a-0850-4cd9-823d-0e3ff82ea125)
NVDA: "error message failed to get chat GPT reply"

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/7c9368ca-26a5-40d0-b96d-8af9ab20ee3d)
NVDA: "information message level progress reset"

## emails
can tab through emails. Tab order goes from oldest email to newest.

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/73b22a24-6322-44b4-869d-8057bce47753)
I notice the phrasing sounds weird with NVDA. It's a bit like
"Email to. bob@example.com subject. Greetings email body. Hello bob."

Great stuff. I think the reading of the email box could be improved. I will make a new ticket for that
